examples/tests/backend_driver.py

```diff
@@ -195,7 +195,6 @@
     'log_test.py',
     'major_minor_demo1.py',
     'major_minor_demo2.py',
-    'manual_axis.py',
```

Comment:
Looks like you didn't actually remove manual_axis.py.  I unticked the box in #7956.